### ISSUE TITLE
Uncaught TypeError: Cannot read property 'default' of undefined

### ISSUE THREAD

**Issue by Ph0enixKM** [sindrets/atom-title-bar-replacer#21]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.23.1 x64
**Electron**: 1.6.15
**OS**: Microsoft Windows 10 Home
**Thrown From**: [title-bar-replacer](https://github.com/sindrets/atom-title-bar-replacer) package 1.7.0

### Stack Trace

Uncaught TypeError: Cannot read property 'default' of undefined

```
At C:\Users\Paweł\.atom\packages\title-bar-replacer\lib\title-bar-replacer.ts:250

TypeError: Cannot read property 'default' of undefined
    at TitleBarReplacer.resetSettings (/packages/title-bar-replacer/lib/title-bar-replacer.ts:250:95)
    at /packages/title-bar-replacer/lib/title-bar-replacer.ts:153:24
    at ~/AppData/Local/atom/app-1.23.1/resources/app/src/config.js:693:26
    at Function.module.exports.Emitter.simpleDispatch (~/AppData/Local/atom/app-1.23.1/resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (~/AppData/Local/atom/app-1.23.1/resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at Config.module.exports.Config.emitChangeEvent (~/AppData/Local/atom/app-1.23.1/resources/app/src/config.js:884:35)
    at Config.module.exports.Config.setRawValue (~/AppData/Local/atom/app-1.23.1/resources/app/src/config.js:670:25)
    at Config.module.exports.Config.set (~/AppData/Local/atom/app-1.23.1/resources/app/src/config.js:248:20)
    at SettingsPanel.set (~/AppData/Local/atom/app-1.23.1/resources/app/node_modules/settings-view/lib/settings-panel.js:237:27)
    at HTMLInputElement.changeHandler (~/AppData/Local/atom/app-1.23.1/resources/app/node_modules/settings-view/lib/settings-panel.js:174:25)
```

### Commands

```
```

### Non-Core Packages

```
atom-babel-compiler 2.3.1 
atom-beautify 0.30.9 
atom-jade 0.3.0 
atom-live-server 2.2.0 
atom-minify 0.8.0 
atom-python-run 0.9.5 
autocomplete-python 1.10.5 
build 0.69.0 
busy-signal 1.4.3 
file-icons 2.1.15 
gpp-compiler 3.0.7 
Hydrogen 2.1.1 
jade 1.2.0 
jade-autocompile 0.8.5 
language-babel 2.81.0 
language-glsl 2.0.4 
minimap 4.29.7 
pigments 0.40.2 
predawn-syntax 1.0.4 
predawn-ui 1.0.7 
sass-autocompile 0.13.3 
title-bar-replacer 1.7.0 
```

**Comment by sindrets** [comment]
@Ph0enixKM thank you for reporting. I've known about this bug for a while, but I only recently started working on this again. Due to the function not being critical to the operation of title bar replacer, I'll wait to patch it until the next bigger update (it's almost ready, but I might not be able to finish it until after Christmas).

**Comment by Ph0enixKM** [comment]
No problem... This is just an error wich shows up whenever I want to install the addon twice (this was by accident), but It isn't really charming or annoying... :)
Regards,
Phoenix

**Comment by sindrets** [comment]
This is now fixed in v1.8.0
